compiler/semantic/op.go

```diff
@@ -410,19 +411,25 @@ func (a *analyzer) semOp(o ast.Op, seq dag.Seq) (dag.Seq, error) {
 	case *ast.From:
 		return a.semFrom(o, seq)
 	case *ast.Summarize:
-		keys, err := a.semAssignments(o.Keys, true)
+		keys, err := a.semAssignments(o.Keys)
 		if err != nil {
 			return nil, err
 		}
+		if assignmentHasDynamicLHS(keys) {
+			return nil, errors.New("summarize: key output fields must be static")
```

Comment:
Make this consistent with the other assignmentHasDynamicLHS error string below.
```suggestion
			return nil, errors.New("summarize: key output field must be static")
```